source/rst/continuous_time/seir_model_sde.rst

```diff
@@ -0,0 +1,922 @@
+
+
+We then look at extending the model to include policy-relevant aggregate shocks, and
+examine the three main techniques for including stochasticity to continuous-time models:
+* Brownian Motion:  A diffusion process with  stochastic, continuous paths.  The prototypical  Stochastic Differential Equation (SDE) with additive noise.
+* Pure-Jump Processes: A variable that jumps between a discrete number of values, typically with a Poisson arrival rate.
+* Jump-Diffusion Process: A stochastic process that contains both a diffusion term and arrival rates of discrete jumps.
+
+Setup
+------------------
+
+.. literalinclude:: /_static/includes/deps_generic.jl
+     :class: hide-output
+
+.. code-block:: julia
+
+    using LinearAlgebra, Statistics, Random, SparseArrays
+
+.. code-block:: julia
+    :class: Test
+
+    using Test # Put this before any code in the lecture.
+
+In addition, we will be exploring packages within the `SciML ecosystem <https://github.com/SciML/>`__ and
+others covered in previous lectures
+
+.. code-block:: julia
+
+    using OrdinaryDiffEq, StochasticDiffEq, DiffEqJump
+    using Parameters, StaticArrays, Plots, RecursiveArrayTools
+
+
+The SEIR Model
+==============
+
+In the version of the SEIR model we will analyze there are four states.
+
+All individuals in the population are assumed to be in one of these four states.
+
+The states are: susceptible (S), exposed (E), infected (I) and removed (R).
+
+Comments:
+
+* Those in state R have been infected and either recovered or died.
+
+* Those who have recovered are assumed to have acquired immunity.
+
+* Those in the exposed group are not yet infectious.
+
+
+
+The interest is primarily in
+
+* the number of infections at a given time (which determines whether or not the health care system is overwhelmed) and
+* how long the caseload can be deferred (hopefully until a vaccine arrives)
+* how to model aggregate shocks related to policy, behavior, and medical innovations
+
+
+
+Changes in the Infected State
+-------------------------------
+
+Within the SEIR model, the flow across states follows the path :math:`S \to E \to I \to R`.  Extensions of this model, such as SEIRS, relax lifetime immunity and allow
+transitions from :math:`R \to S`.
+
+The transitions between those states are governed by the following rates
+
+* :math:`\beta(t)` is called the *transmission rate* (the rate at which individuals bump into others and expose them to the virus).
+* :math:`\sigma` is called the *infection rate* (the rate at which those who are exposed become infected)
+* :math:`\gamma` is called the *recovery rate* (the rate at which infected people recover or die).
+
+
+In addition, the transmission rate can be re-parameterized as: :math:`R(t) := \beta(t) / \gamma` where :math:`R(t)` has the interpretation as the *effective reproduction number* at time :math:`t`.  For this reason, we will work with the :math:`R(t)` reparameterization.
```

Comment:
@jstac  I took this from the python lecture, but I am not sure it is correct in epidemilogy lingo.  This might be the R_0 instead (i.e. the "Basic reproduction number") instead.  See https://en.wikipedia.org/wiki/Basic_reproduction_number and other stuff.

But understanding from this is that the base reproduction number is that where we make everyone susceptible, while the effective reproduction number takes into account the current equilibrium?

Reply:
I believe in your implementation you may talk about `R0` instead? https://python.quantecon.org/sir_model.html#Implementation

Reply:
Finally, for the "effective reproduction rate", which might actually be useful to plot as well, I think it might be simply `R_e(t) = R_0 * s(t)`  i.e. the derivative of the  `R_0 * I/N * S` flow wrt `I`.  Or the marginal number of new infections for an increase in the number of infected?

Reply:
@jstac   I implemented these sorts of changes in the latest version, and there are a significant number of changes along these topics.